### PR TITLE
Revise Fusion content

### PR DESCRIPTION
**Pull request by jason-seqera**
- https://seqera.atlassian.net/browse/EDU-253

Pages:
- https://docs.seqera.io/fusion/guide/ssd
- https://docs.seqera.io/fusion/troubleshooting
- https://docs.seqera.io/fusion/faq

### CODE REVIEW

File: fusion_docs/faq.mdx [hunk 1]
```diff
@@ -4,30 +4,27 @@ title: FAQ
...
+**Wave containers as a prerequisite for using Fusion**
 
-This would require to rebuild all containers used by your data pipeline to include the Fusion client each time a new
-version of the Fusion client is released, and it would make necessary to maintain a custom mirror or existing containers
-collections, such as [BioContainers](https://biocontainers.pro/) which is definitively not desirable.
+Fusion works at the level of Netflow job execution. For this reason, it needs to run in containerized job execution context.
 
-Wave allows adding the Fusion client in your pipeline containers at deploy time, without having to rebuild them or
-to maintainer a separate container images collection.
+This would require to rebuild all containers used by your data pipeline to include the Fusion client each time a new version of the Fusion client is released, and it would make necessary to maintain a custom mirror or existing containers collections, such as [BioContainers](https://biocontainers.pro/) which is definitively not desirable.
```
Comment (justinegeffen) [inline fusion_docs/faq.mdx:25]:
```suggestion
When a new Fusion client version is released, you'll need to rebuild all the containers used by your data pipeline to include the new version. This makes it necessary to maintain a custom mirror or existing container collections, such as [BioContainers](https://biocontainers.pro/) which is not recommended.
```

File: fusion_docs/faq.mdx [hunk 2]
```diff
@@ -4,30 +4,27 @@ title: FAQ
...
+Currently, Fusion can be used only by enabling Wave containers in the configuration of your Nextflow pipeline. Fusion is not available as a separate download.
 
-Fusion is designed to work at level of job executions. For this reason, it needs to run in containerised job
-execution context.
+**Wave containers as a prerequisite for using Fusion**
 
-This would require to rebuild all containers used by your data pipeline to include the Fusion client each time a new
-version of the Fusion client is released, and it would make necessary to maintain a custom mirror or existing containers
-collections, such as [BioContainers](https://biocontainers.pro/) which is definitively not desirable.
+Fusion works at the level of Netflow job execution. For this reason, it needs to run in containerized job execution context.
```
Comment (justinegeffen) [inline fusion_docs/faq.mdx:23]:
```suggestion
Fusion works at the level of Nextflow job execution. For this reason, it needs to run in a containerized job execution context.
```

File: fusion_docs/faq.mdx [hunk 3]
```diff
@@ -4,30 +4,27 @@ title: FAQ
...
 
-Fusion currently supports AWS S3 and Google Storage. In the near future it will also support Azure Blob storage.
+Fusion currently supports AWS S3, Azure Blob Storage, and Google Cloud Storage.
 
-## Can I use Fusion with Minio?
+**Fusion support for MinIO**
 
-Yes. [Minio](https://min.io/), implements a S3-compatible API, therefore it can be used in place of AWS S3.
+[MinIO](https://min.io/), which implements an S3-compatible API, is supported in place of AWS S3.
 See the documentation how to configure your pipeline execution to use Fusion and Minio. (link to guide TBD).
```
Comment (justinegeffen) [inline fusion_docs/faq.mdx:14]:
```suggestion
See the documentation how to configure your pipeline execution to use Fusion and MinIO. (link to guide TBD).
```

File: fusion_docs/faq.mdx [hunk 4]
```diff
@@ -4,30 +4,27 @@ title: FAQ
...
-This would require to rebuild all containers used by your data pipeline to include the Fusion client each time a new
-version of the Fusion client is released, and it would make necessary to maintain a custom mirror or existing containers
-collections, such as [BioContainers](https://biocontainers.pro/) which is definitively not desirable.
+Fusion works at the level of Netflow job execution. For this reason, it needs to run in containerized job execution context.
 
-Wave allows adding the Fusion client in your pipeline containers at deploy time, without having to rebuild them or
-to maintainer a separate container images collection.
+This would require to rebuild all containers used by your data pipeline to include the Fusion client each time a new version of the Fusion client is released, and it would make necessary to maintain a custom mirror or existing containers collections, such as [BioContainers](https://biocontainers.pro/) which is definitively not desirable.
+
+Wave allows adding the Fusion client in your pipeline containers at deploy time, without having to rebuild them or to maintainer a separate container image collection.
```
Comment (justinegeffen) [inline fusion_docs/faq.mdx:27]:
```suggestion
Wave allows adding the Fusion client in your pipeline containers at deploy time, without having to rebuild them or to maintaine a separate container image collection.
```

**Comment by justinegeffen** [comment]
@llewellyn-sl is this content part of your Fusion audit? If so, I think we can close this PR.

File: fusion_docs/faq.mdx [hunk 5]
```diff
@@ -4,30 +4,27 @@ title: FAQ
...
+**Fusion support for MinIO**
 
-Yes. [Minio](https://min.io/), implements a S3-compatible API, therefore it can be used in place of AWS S3.
+[MinIO](https://min.io/), which implements an S3-compatible API, is supported in place of AWS S3.
 See the documentation how to configure your pipeline execution to use Fusion and Minio. (link to guide TBD).
 
-## Can I download Fusion?
 
-No. Currently, Fusion can only be used by enabling Wave containers in the configuration of your Nextflow pipeline.
+**Fusion as a separate download**
```
Comment (pditommaso) [inline fusion_docs/faq.mdx:17]: By definition a FAQ is organised in question

**Comment by justinegeffen** [comment]
I'm closing this in favor of @llewellyn-sl's work on the Fusion docs. Thanks for starting this, @jason-seqera!

**Pull request closed without merging**